build-logic-commons/module-identity/src/main/kotlin/gradlebuild/identity/extension/ReleasedVersionsDetails.kt

```diff
@@ -34,7 +34,7 @@ class ReleasedVersionsDetails(currentBaseVersion: GradleVersion, releasedVersion
 
     init {
         val lowestInterestingVersion = GradleVersion.version("0.8")
-        val lowestTestedVersion = GradleVersion.version("1.0")
```

Comment:
This is the lowest version for the target distribution, right?

Reply:
both to my understanding, with this we would still run TAPI clients that are lower than 7.0.
running `gradle3.0CrossVersionTest` means running everything where 3.0 is involved also the 3.0 clients if not restricted.